### DISCUSSION TITLE
Trophy for earning Achievement of the Week

### DISCUSSION THREAD

**Discussion by Jamiras** [RetroAchievements/RAWeb#740] [Ideas]
As a player, I'd like to know how many "Achievement of the Week"s that I have completed. I would also like to receive a trophy for completing multiple "Achievement of the Week"s.

The new trophy would be added to the user's trophy list in a manner similar to the developer trophy. The tooltip would display how many AotWs the user has completed. Clicking on the trophy would display a list of historical AotWs, using grayed out icons to illustrate which ones the player didn't earn.

**Comment by coolscribble** [comment]
I think it's not too important like this, but we can make "stat" where can be written all achievements of the week he won, something like "View all achievements sets xxx has worked on."

**Comment by Jamiras** [comment]
Tiered AotW trophies have been around for a while, but they're associated with the number of achievements earned during a year. #2778 allows event achievements to attach to actual achievements and track player progress for a given year.
